src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```diff
@@ -17,18 +17,23 @@ parameters:
   ubuntu1804Container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04
   poolInternalAmd64:
     name: NetCore1ESPool-Svc-Internal
-    demands: ImageOverride -equals Build.Ubuntu.1804.Amd64
+    image: 1es-ubuntu-2004
+    os: linux
   poolInternalAmd64PR:
     name: NetCore1ESPool-Internal-XL
-    demands: ImageOverride -equals Build.Ubuntu.1804.Amd64
+    image: 1es-ubuntu-2004
+    os: linux
   poolInternalArm64:
     name: Docker-Linux-Arm-Internal
+    image: Mariner-2-Docker-ARM64
+    os: linux
+    hostArchitecture: Arm64
   poolPublicAmd64:
     name: NetCore-Public-XL
-    demands: ImageOverride -equals Build.Ubuntu.1804.Amd64.Open
-
+    image: 1es-ubuntu-2004-open
+    os: linux
 jobs:
-- template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-build-tarball.yml
+- template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-build-tarball.yml@self
```

Comment:
It appears as though this is running on a windows machine.  e.g. "C:\Windows\system32\cmd.exe" /D /E:ON /V:OFF /S /C "CALL "D:\a\_work\_temp\eecb05cc-8daa-4806-9ce3-b080361c6ec5.cmd"" which would explain the CI failures.